### PR TITLE
split datasets and simulators, update jaer links

### PR DESCRIPTION
**Pull request by tobidelbruck**
See how this looks... maybe easier to navigate?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,19 +92,19 @@ ___
 # Survey paper
 - <a name="Gallego20tpami"></a>Gallego, G., Delbruck, T., Orchard, G., Bartolozzi, C., Taba, B., Censi, A., Leutenegger, S., Davison, A., Conradt, J., Daniilidis, K., Scaramuzza, D.,  
 **_[Event-based Vision: A Survey](http://rpg.ifi.uzh.ch/docs/EventVisionSurvey.pdf)_**,  
-IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 44(1):154-180, Jan. 2022.
+IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 2020.
```
Comment (guillermogb) [inline README.md:95]: please revert this line. It was good before.
44(1):154-180, Jan. 2022.

File: README.md [hunk 2]
```diff
@@ -91,19 +92,19 @@ ___
...
 
 
 <a name="workshops"></a>
 # Workshops
 - [CVPR 2023 Fourth International Workshop on Event-based Vision](https://tub-rip.github.io/eventvision2023), **[Videos](https://www.youtube.com/playlist?list=PLeXWz-g2If96iotpzgBNNTr9VA6hG-LLK)**
-- [IEEE Embedded Vision Workshop Series](https://embeddedvisionworkshop.wordpress.com), with focus on Biologically-inspired vision and embedded systems.
```
Comment (guillermogb) [inline README.md:100]: please keep this line. Revert

File: README.md [hunk 3]
```diff
@@ -128,28 +129,26 @@ IEEE Trans. Pattern Anal. Machine Intell. (TPAMI), 44(1):154-180, Jan. 2022.
...
-        - Prototype only 
-    - **SDAVIS192**: Moeys, D. P., Corradi, F., Li, C., Bamford, S. A., Longinotti, L., Voigt, F. F., Berry, S., Taverni, G., Helmchen, F., Delbruck, T., *[A Sensitive Dynamic and Active Pixel Vision Sensor for Color or Neural Imaging Applications](https://doi.org/10.1109/TBCAS.2017.2759783)*, IEEE Trans. Biomed. Circuits Syst. 12(1):123-136 2018.
-        - Prototype only
-    - **Omnivision HVS**: Guo et al, [A 3-Wafer-Stacked Hybrid 15MPixel CIS + 1 MPixel EVS with 4.6GEvent/s Readout, In-Pixel TDC and On-Chip ISP and ESP Function](http://dx.doi.org/10.1109/isscc42615.2023.10067476), ISSCC, (2023).
-        - Prototype, commercially n.a. 
-    - **Sony HVS**: Kodama et al., [1.22μm 35.6Mpixel RGB Hybrid Event-Based Vision Sensor with 4.88μm-Pitch Event Pixels and up to 10K Event Frame Rate by Adaptive Control on Event Sparsity](http://dx.doi.org/10.1109/ISSCC42615.2023.10067520), ISSCC (2023)
-        - Prototype only, commercially n.a. 
+- **HVS** (Hybrid Vision Sensors) like  **DAVIS**, **CDAVIS**, and other HVS that have brightness change events and intensity frames, either mono or color
```
Comment (guillermogb) [inline README.md:132]: It seems that previous PRs are being discarded.
Please revert. This section should not change:
image

**Review by guillermogb — CHANGES_REQUESTED** [review]

**Comment by tobidelbruck** [comment]
Sorry about this, I admit that I'm clueless still about pull requests and dealing with commits that happen during pull. Let me try to figure it out. I must have started with a version that was not the latest master.

**Comment by tobidelbruck** [comment]
I hope I managed to merge everything correctly.

**Comment by guillermogb** [comment]
Thank you Tobi. Some code was overwriting previous pull requests. I think I merged the new part from this PR in two commits ccb4619aa64ea65aa0f2c24705905eb927f4e5a9 and 5b132a63eb80665970bfab6ce70df475f6e4b148 
Please double check. Otherwise, we open a new PR.

**Pull request closed without merging**